packages/audits/src/components/AuditsTableCell.tsx

```diff
@@ -18,6 +20,29 @@ const AuditsTag: CellComponent = (props) => {
   );
 };
 
+// Logos take from the cdnjs site. The repository with the src is here https://github.com/alrra/browser-logos
+const UserAgent: CellComponent = ({ value }) => {
+  const browser = useMemo(() => {
+    const browser = new UAParser(value).getBrowser().name;
+    return browser && browsers.includes(browser.toLowerCase() ?? '') ? browser : 'Web';
+  }, [value]);
+
+  return value ? (
+    <div className={classNames('fe-audits__severity')}>
+      <Popup
+        action='hover'
+        content={<div className={`${prefixCls}__useragent-hover`}>{value}</div>}
+        trigger={
+          <img
+            alt={browser}
+            src={`//cdnjs.cloudflare.com/ajax/libs/browser-logos/69.0.4/${browser.toLowerCase()}/${browser.toLowerCase()}_${sizeOfIcon}x${sizeOfIcon}.png`}
```

Comment:
calculate the url outside the JSX code

Reply:
I don't see the reason to do it. But I'll do it.

Reply:
It's just long lines in JSX code